app/lib/document_types/F13614CF150802020.rb

```diff
@@ -0,0 +1,19 @@
+module DocumentTypes
+  class F13614CF150802020 < DocumentType
```

Comment:
This is a super funky class name, but there is some precedent to using the numbered file name as the document type... Seemed like it would be possibly easier to debug than using a key that did not reflect the class name.

Reply:
suggest `Form13614CForm15080`